app/src/playstore/java/tice/AppFlowExtensions.kt

```diff
@@ -17,3 +20,10 @@ suspend fun AppFlow.updatePushDeviceId() {
         }
     }?.let { verifyDeviceHandler.get().startUpdatingDeviceId(it) }
 }
+fun AppFlow.storeSpecificSetup(context: Context) {
+    if (BuildFlavorStore.PLAY_STORE.gmsAvailable(application.applicationContext)) {
```

Comment:
Das hätte ich hier so nicht erwartet. Wenn gms nicht available ist, wird Mapbox initialisiert? Was ist dann hier mit FDroid?

Reply:
Dieser Code ist nur in der Playstore-Variante vorhanden.